worlds/timespinner/docs/en_Timespinner.md

```diff
@@ -21,6 +21,8 @@ The goal remains unchanged. Kill the Sandman\Nightmare!
 All main inventory items, orbs, collectables, and familiars can be shuffled, and all locations in the game which could
 contain any of those items may have their contents changed.
 
+In total, there are **166 items** to find.
```

Comment:
What is this count? total on unique items? or Number of checks?

if its unique items i dont see a lot of value in stating that here as many items never get send by the server , they are just there incase someone wants it in thier starting inventory.

If its number of checks, it greatly varies depending on options and it will likely be a number people forget to update each time they add new checks.

If this is something we will push through for all games, sure. but in general i am not in favour of this change

Reply:
It's the number of checks, yes. If we can get more specific with the possible range that would be even better, but I think it's still very useful to have a ballpark number that represents the default settings. We can couch it in looser language if you prefer, like: 

> With the default YAML, there are 166 items to find, but this number can vary depending on your settings.

Reply:
i just checked on latest RC, with default settings its 180, with max settings its 190 and lowest is 166